app/api/posts/$$.mjs

```diff
@@ -1,8 +1,11 @@
+import { dirname, join } from 'path'
```

Comment:
since we're using Node 18, we could use the new colon prefix `node:path`. not required, but there are security benefits